backend/src/cache/mutation.go

```diff
@@ -19,16 +19,20 @@ import (
 	"encoding/hex"
 	"fmt"
 	"log"
+	"strings"
 
 	"k8s.io/api/admission/v1beta1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
+	KFPAnnotation        string = "pipelines.kubeflow.org"
+	ArgoWorkflowNodeName string = "workflows.argoproj.io/node-name"
 	ArgoWorkflowTemplate string = "workflows.argoproj.io/template"
 	ExecutionKey         string = "pipelines.kubeflow.org/execution_cache_key"
 	AnnotationPath       string = "/metadata/annotations"
+	TFXPodSuffix         string = "tfx/orchestration/kubeflow/container_entrypoint.py"
```

Comment:
I know we don't have a significantly better alternative to this yet. Just curious if that's feasible if we filter the pod by label [1] if we add some special labels for TFX pods?

[1] https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

Reply:
Good point. I think this is doable and can be improved later on. +cc @Ark-kun